firecracker-control/cmd/containerd/main.go

```diff
@@ -46,6 +46,10 @@ import (
 	_ "github.com/containerd/containerd/runtime/v2"
 	_ "github.com/containerd/containerd/runtime/v2/runc/options"
 
+	// Snapshotters
+	_ "github.com/containerd/containerd/snapshots/devmapper"
+	_ "github.com/containerd/containerd/snapshots/overlay"
```

Comment:
Do we need overlay?

Reply:
It's convenient to have it during development, when you need to test something quickly and you don't have thin-pool prepared.